### PR TITLE
fit to layer extent

### PR DESCRIPTION
**Pull request by taulinger**
Add the possibility to fit on a specific vector layer on the map
A fit request is resolved by the map implementation.
Fit requests for other layer types may be implemented in the future.

### CODE REVIEW

File: src/modules/olMap/components/OlMap.js [hunk 1]
```diff
@@ -187,19 +188,23 @@ export class OlMap extends MvuElement {
...
 			};
-			const maxZoom = fitRequest.payload.options.maxZoom || this._view.getMaxZoom();
-			this._view.fit(fitRequest.payload.extent, { maxZoom: maxZoom, callback: onAfterFit });
+			const extent = eventLike.payload.id ? getLayerById(this._map, eventLike.payload.id).getSource()?.getExtent() : eventLike.payload.extent;
+
+			if (extent) {
+				this._viewSyncBlocked = true;
+				const maxZoom = eventLike.payload.options.maxZoom || this._view.getMaxZoom();
+				this._view.fit(extent, { maxZoom: maxZoom, callback: onAfterFit }); // Todo: fit map with padding parameter by calling mapService#calculatePadding
+			}
```
Comment (thiloSchlemmer) [inline src/modules/olMap/components/OlMap.js:202]: Can we extract this to a helper-method, to be consistent to the code unterneath?

**Review by thiloSchlemmer — APPROVED** [review]
Nice! One small hint inside

File: src/modules/olMap/components/OlMap.js (hunk 1 above)
Comment (taulinger) [reply src/modules/olMap/components/OlMap.js:202]: >to be consistent to the code unterneath?

 Sorry, didn't get the point, can you explain it more in detail, please?

Comment (thiloSchlemmer) [reply src/modules/olMap/components/OlMap.js:202]: sure...

Line 205 and 207 uses only simple calls to private functions,  the code in line 192 - 202 can be moved to a simple method, too. In the result, the reader gets a consistent overview over all observed properties and the related behavior.

**Review by thiloSchlemmer — APPROVED** [review]
Nice, THX!

**Pull request merged**